### PR TITLE
v1/card : Add Card definition

### PR DESCRIPTION
**Pull request by nkristianto**
Add API definition for card transaction. This reffered to ACB protobuf
definition for Card.
https://gitlab.dev.acb.com.vn/acb/acb-proto/blob/master/pb/v1/card/card.proto

Updates [OB-34](https://brankas.atlassian.net/browse/OB-34)

### CODE REVIEW

File: v1/card/card.gunk [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+//                 openapiv2.HTTPS,
+//         },
+//         Consumes: []string{
+//                 "application/json",
+//         },
+//         Produces: []string{
+//                 "application/json",
+//         },
+//         Responses: map[string]openapiv2.Response{
+//                 "400": openapiv2.Response{
```
Comment (cuonglm) [inline v1/card/card.gunk:19]: Add 404 please

Comment (nkristianto) [reply v1/card/card.gunk:19]: Thanks. Updated.
PTAL @cuonglm

File: v1/card/card.gunk [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+	types "github.com/openbank/openbank/v1/types"
+)
+
+// Card holds all details about a card.
+type Card struct {
+	// ID is a unique identifier of a card.
+	ID string `pb:"1" json:"id"`
+
+	// Account is the account id for the card.
```
Comment (cuonglm) [inline v1/card/card.gunk]: Maybe better to change to `account id who owns the card` or `account id of the card`.

**Review by cuonglm — APPROVED** [review]

**Comment by cuonglm** [comment]
Just comestic change  in comment, otherwise, LGTM.

**Review by Hekewa — APPROVED** [review]
:rocket:

**Pull request merged**